lens/lotusrepo/repo.go

```diff
@@ -111,6 +112,10 @@ type RepoAPI struct {
 	cacheSize int
 }
 
+func (ra *RepoAPI) ChainGetTipSetAfterHeight(ctx context.Context, epoch abi.ChainEpoch, key types.TipSetKey) (*types.TipSet, error) {
+	panic("implement me")
```

Comment:
See my comment elsewhere. We could fudge it by calling ChainGetTipSetByHeight(height+1). This repo lens is still in current use for backfilling so we can't not support it right now.

Reply:
I just reimplemented what I did for the lily impl, should work correctly for null rounds now as well.